server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java

```diff
@@ -342,7 +347,7 @@ private boolean isRetryTimeout(long now, long timeout, long beginTime) {
      */
     protected void handleAsyncCommitting() {
         Collection<GlobalSession> asyncCommittingSessions = SessionHolder.getAsyncCommittingSessionManager()
-            .allSessions();
+                .allSessions();
```

Comment:
Please do not change the format if do not change the code.

Reply:
solved